From 39c8ed6d7ef79fefdc3b5906428ff41edf6ec1b4 Mon Sep 17 00:00:00 2001
From: Razvan Stefanescu <razvan.stefanescu@nxp.com>
Date: Thu, 12 Apr 2018 15:55:30 +0300
Subject: [PATCH 524/706] staging: dpaa2-evb: Fix MC bus include

Adapt to upstream changes.

Signed-off-by: Razvan Stefanescu <razvan.stefanescu@nxp.com>
(cherry picked from commit 9605f1dfd8623c3390de72406952d8a11c6dd710)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/staging/fsl-dpaa2/evb/dpdmux.c | 2 +-
 drivers/staging/fsl-dpaa2/evb/evb.c    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/evb/dpdmux.c b/drivers/staging/fsl-dpaa2/evb/dpdmux.c
index 7fd8415d497a..4c4a059e726c 100644
--- a/drivers/staging/fsl-dpaa2/evb/dpdmux.c
+++ b/drivers/staging/fsl-dpaa2/evb/dpdmux.c
@@ -29,7 +29,7 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-#include "../../fsl-mc/include/mc.h"
+#include <linux/fsl/mc.h>
 #include "dpdmux.h"
 #include "dpdmux-cmd.h"
 
diff --git a/drivers/staging/fsl-dpaa2/evb/evb.c b/drivers/staging/fsl-dpaa2/evb/evb.c
index 219c4758d316..86037640a211 100644
--- a/drivers/staging/fsl-dpaa2/evb/evb.c
+++ b/drivers/staging/fsl-dpaa2/evb/evb.c
@@ -38,7 +38,7 @@
 #include <uapi/linux/if_bridge.h>
 #include <net/netlink.h>
 
-#include "../../fsl-mc/include/mc.h"
+#include <linux/fsl/mc.h>
 
 #include "dpdmux.h"
 #include "dpdmux-cmd.h"
-- 
2.17.1

